### PR TITLE
Global styles

### PR DESCRIPTION
**Pull request by SolivanLau**
**Worked on:**
- global styles (variables, links,  buttons, form elements, components, etc.)
- normalize is in
- head and footer html/css
- imported global svg assets (mostly logo, blockquotes, socials, menu icons)
- added basic modal functionality and styles 
- added HTML markup for about and values
- added mobile menu

**Next steps**
- responsive styling for footer and nav
- discuss best solution  for overlapping components

### CODE REVIEW

File: assets/icons/global/logo.svg
Comment (maxharrisnet) [inline assets/icons/global/logo.svg:1]: Can we rename this to `itlct-logo.svg` please?

File: css/global.css [hunk 1]
```diff
@@ -17,9 +17,203 @@ Resets (box-sizing, normalize, etc.)
 ===================================
 */
 
+/* box-sizing */
 * {
 	box-sizing: border-box;
 }
+/* 
+minified css normalize from Nicolas Gallagher:
```
Comment (maxharrisnet) [inline css/global.css:25]: @SolivanLau  I think Prettier is reversing your magnification here. Try using Prettier's ignore feature: https://prettier.io/docs/en/ignore.html

Comment (maxharrisnet) [reply css/global.css:25]: Perfect way to include it, btw 👍🏾

File: css/global.css [hunk 2]
```diff
@@ -30,15 +224,48 @@ Variables (colors, text, spacing, etc.)
...
+	--color-teal: #007a80;
+	--color-grape: #6c4681;
+	--color-onyx: #1f1d1d;
+	--color-blue: #184f85;
+	--color-pink: #fa6cb7;
+	--color-citrine: #d3c53f;
+	--color-citron: #92a32d;
+	--color-copper: #b37122;
 
 	/* Color Assignment */
```
Comment (maxharrisnet) [inline css/global.css:237]: We'll need to add a couple of variables for text color (once we have backgrounds for white text) and apply them.

File: css/global.css [hunk 3]
```diff
@@ -30,15 +224,48 @@ Variables (colors, text, spacing, etc.)
...
 	--body-background-color: var(--color-white);
 
 	/* Text */
+	/* Font */
+	--font-main: "Josefin Sans", sans-serif;
+	/* Sizing */
+	/* heading elements */
```
Comment (maxharrisnet) [inline css/global.css:245]: This looks really good, thanks for the comments

File: css/global.css [hunk 4]
```diff
@@ -30,15 +224,48 @@ Variables (colors, text, spacing, etc.)
...
+	--spacing-lg: 80px;
+	--spacing-med: 40px;
+	--spacing-sm: 20px;
 
+	--border-radius: 10px;
 	/* Wrappers */
+	--wrapper-width: 85%;
+	--wrapper-max-width: 1328px;
 
 	/* Media Queries */
```
Comment (maxharrisnet) [inline css/global.css:280]: Let's add the different widths for our media queries here with something like:
```
--mq-width-tablet-max:  768px;
--mq-width-phone-max:  400px;
```

And let's replace the values for the media queries after line 698

Comment (maxharrisnet) [reply css/global.css:280]: This will help us a lot when  we are working on different pages.

File: css/global.css [hunk 5]
```diff
@@ -63,33 +292,353 @@ img {
...
 
 /* 
 ===================================
 Global Component Styles (buttons, forms, etc.)
 ===================================
 */
+/* Section */
+
+section {
+	margin: var(--spacing-lg) 0;
```
Comment (maxharrisnet) [inline css/global.css]: We'll want to use padding here instead of margin so that backgrounds will work

File: css/global.css [hunk 6]
```diff
@@ -63,33 +292,353 @@ img {
...
+hr {
+	border-color: var(--color-white);
+}
+
+blockquote {
+	background-color: var(--color-onyx);
+	color: var(--color-white);
+
+	width: fit-content;
+	padding: 80px;
```
Comment (maxharrisnet) [inline css/global.css]: Can we use a variable here?

Comment (maxharrisnet) [reply css/global.css]: Actually, there are a number of places below this line where we can replace values with variables. Looks like all of them are related to spacing (gap, padding, top, left, etc.)

File: css/global.css [hunk 7]
```diff
@@ -30,15 +224,48 @@ Variables (colors, text, spacing, etc.)
...
+	--font-weight-med: 500;
+	--font-weight-reg: 400;
 
 	/* Spacing */
+	--spacing-xl: 100px;
+	--spacing-lg: 80px;
+	--spacing-med: 40px;
+	--spacing-sm: 20px;
 
+	--border-radius: 10px;
```
Comment (maxharrisnet) [inline css/global.css:274]: We should add border widths here as well. I forgot to comment out a section for borders.

File: assets/icons/global/logo.svg
Comment (SolivanLau) [reply assets/icons/global/logo.svg:1]: Sounds good! edit has been made!

File: css/global.css (hunk 5 above)
Comment (SolivanLau) [reply css/global.css]: yes! Just corrected it!

File: css/global.css (hunk 7 above)
Comment (SolivanLau) [reply css/global.css:274]: made 2 so far - links and buttons

File: css/global.css (hunk 6 above)
Comment (SolivanLau) [reply css/global.css]: yes! I noticed that there are some magic numbers added outside of the 20-40-100 rule in the design. Will adjust as many values that fit w 20-40-100 rule now and add others later!

File: css/global.css (hunk 1 above)
Comment (SolivanLau) [reply css/global.css:25]: will check on this - might need to touch base w you to double check!

File: css/global.css (hunk 2 above)
Comment (SolivanLau) [reply css/global.css:237]: will make variables for light and dark color text!

**Review by SolivanLau — COMMENTED** [review]
apart from prettier config, all requested changes have been made!

File: css/global.css (hunk 6 above)
Comment (maxharrisnet) [reply css/global.css]: Sounds good. If there's a magic value that's only used once, we don't need to make a variable for it.

File: css/global.css (hunk 7 above)
Comment (maxharrisnet) [reply css/global.css:274]: Perfect!

File: css/global.css (hunk 5 above)
Comment (maxharrisnet) [reply css/global.css]: Looks good!

**Review by maxharrisnet — APPROVED** [review]
@SolivanLau thanks for addressing the changes. We can fix prettier/normalize in another PR. Looks good to me!

File: css/page-template.css [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
```
Comment (maxharrisnet) [inline css/page-template.css:38]: Don't forget to switch this out for the desktop-first version with the variables.

File: css/global.css [hunk 9]
```diff
@@ -30,17 +224,63 @@ Variables (colors, text, spacing, etc.)
...
+	--font-size-body-lg: 2.1rem; /* Body Large - 21px */
+	--font-size-body-reg: 1.6rem; /* Body Regular - 16px */
+
+	/* Text Weight */
+	--font-weight-bold: 700;
+	--font-weight-med: 500;
+	--font-weight-reg: 400;
+
+	/* Line Height */
+	--line-height-lg: 26px;
```
Comment (maxharrisnet) [inline css/global.css:261]: I think the line-heights should be in rems

**Review by maxharrisnet — APPROVED** [review]
We'll work through the homepage stuff in another PR. Looks good!

**Pull request merged**